src/server.ts

```diff
@@ -1,12 +1,13 @@
-import { type Request, type Response, type Express } from "express"
-import { type Logger } from "winston"
-import { port } from "./config/dbConfig"
-import express from "express"
-import bodyParser from "body-parser"
-import loggerManager from "./utility/logger"
-import router from "./routes/courseRoutes"
-import Database from "./db/dbConnection"
-import errorEnums from "./constants/errorConstants"
+import { type Request, type Response, type Express } from "express";
+import { type Logger } from "winston";
+import { port } from "./config/dbConfig";
+import express from "express";
+import bodyParser from "body-parser";
+import loggerManager from "./utility/logger";
+import router from "./routes/courseRoutes";
+import Database from "./db/dbConnection";
+import errorEnums from "./constants/errorConstants";
```

Comment:
You are just adding a semicolon to these lines. Let's put these into a separate PR for linter